Build Failure with MinGW-w64 and GCC 9.1.0 on Windows

Using the following configuration in CMake:

```shell
cmake ../src -G "Unix Makefiles" \
-DBUILD_SHARED_LIBS=OFF -DOPTION_BUILD_TESTS=OFF -DOPTION_BUILD_TOOLS=OFF \
"-DCMAKE_CXX_FLAGS=-DBOOST_THREAD_VERSION=4 -O3 -s -DSYSTEM_WINDOWS" \
"-DCMAKE_INSTALL_PREFIX=$X_WORKDIR_ROOT/dest" \
"-DOPENGL_gl_LIBRARY=$X_DISTRO_ROOT/x86_64-w64-mingw32/lib/libopengl32.a" \
"-DOPENGL_INCLUDE_DIR=$X_DISTRO_ROOT/x86_64-w64-mingw32/include/gl" \
-DOPTION_BUILD_EXAMPLES=OFF -DOPTION_BUILD_WITH_BOOST_THREAD=ON
```

glbinding 3.1.0 fails with the following error:
```
D:/mingw-distro-setup/src/source/glbinding/source/getProcAddress.cpp:33:32: error: invalid conversion from 'FARPROC' {aka 'long long int (*)()'} to 'void*' [-fpermissive]
   33 |  procAddress = ::GetProcAddress(module, name);
      |                ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
      |                                |
      |                                FARPROC {aka long long int (*)()}
make[2]: *** [source/glbinding/CMakeFiles/glbinding.dir/build.make:63: source/glbinding/CMakeFiles/glbinding.dir/source/getProcAddress.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

This is being run on MSYS2 with GCC 9.1.0 and MinGW-w64 7.0.0 (latest commit on master).

--- a/source/glbinding/source/getProcAddress.cpp
+++ b/source/glbinding/source/getProcAddress.cpp
@@ -21,7 +21,7 @@ ProcAddress getProcAddress(const char * name)
     static auto module = LoadLibrary(_T("OPENGL32.DLL"));
 
 	// Prevent static linking of opengl32
-	static auto wglGetProcAddress_ = reinterpret_cast<void * (__stdcall *)(const char *)>(::GetProcAddress(module, "wglGetProcAddress"));
+	static auto wglGetProcAddress_ = reinterpret_cast<void * (__stdcall *)(const char *)>((uintptr_t) ::GetProcAddress(module, "wglGetProcAddress"));
 	assert(wglGetProcAddress_ != nullptr);
 
 	auto procAddress = wglGetProcAddress_(name);
@@ -30,7 +30,7 @@ ProcAddress getProcAddress(const char * name)
 		return reinterpret_cast<ProcAddress>(procAddress);
 	}
 
-	procAddress = ::GetProcAddress(module, name);
+	procAddress = (void *) ::GetProcAddress(module, name);
     return reinterpret_cast<ProcAddress>(procAddress);
 }
 

